editor/app/src/XcodeProjectPatcher.cs

```diff
@@ -96,15 +99,24 @@ internal class XcodeProjectPatcher : AssetPostprocessor {
         ReadConfig(errorOnNoConfig: false);
     }
 
-    // Get the iOS bundle / application ID.
-    private static string GetIOSApplicationId() {
-      return UnityCompat.GetApplicationId(BuildTarget.iOS);
+    // Get the iOS+ bundle / application ID.
+    private static string GetApplicationId() {
+        switch(EditorUserBuildSettings.activeBuildTarget) {
+            case BuildTarget.iOS:
+                return UnityCompat.GetApplicationId(BuildTarget.iOS);
+                break;
+            case BuildTarget.tvOS:
+                return UnityCompat.GetApplicationId(BuildTarget.tvOS);
+                break;
+            default:
+                throw new Exception("unsupported iOS+ version");
```

Comment:
This is a noticeable change of behavior.  Are there any chances that GetApplicationId gets called when the BuildTarget isn't one of those two?  It feels like it would be safer to return tvOS is set to that, otherwise return iOS.

Reply:
Ok. Done. I also changed the method name to `GetIosPlusApplicationId`